target/linux/realtek/dts-5.10/rtl8380_d-link_dgs-1210-10mp.dts

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "rtl838x_d-link_dgs-1210_soc.dtsi"
+#include "rtl83xx_d-link_dgs-1210_poe.dtsi"
+
+/ {
+	compatible = "d-link,dgs-1210-10mp", "realtek,rtl838x-soc";
+	model = "D-Link DGS-1210-10MP";
+};
+
+&ethernet0 {
+	mdio: mdio-bus {
+		compatible = "realtek,rtl838x-mdio";
+		regmap = <&ethernet0>;
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		INTERNAL_PHY(8)
+		INTERNAL_PHY(9)
+		INTERNAL_PHY(10)
+		INTERNAL_PHY(11)
+		INTERNAL_PHY(12)
+		INTERNAL_PHY(13)
+		INTERNAL_PHY(14)
+		INTERNAL_PHY(15)
+		INTERNAL_PHY(24)
+		INTERNAL_PHY(26)
+	};
+};
+
+&switch0 {
+	ports {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		SWITCH_PORT(8, 1, internal)
```

Comment:
as above, it's not bad to be expressive here :)

Reply:
This is also the standard code for every dts file in every realtek device so there is no problem.

Reply:
I understand, I know we do a lot of copy/pasting around here, so this is thus inherent from this. Doesn't make it any better or less ugly :) But this can easily be fixed once it goes upstream, and getting it merged now is better of course.